source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,113 @@
+===========================
+Update a Multiple Documents
+===========================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method on
+a ``MongoCollection`` object. The method accepts a **filter** that matches the
+document you want to update and an **update** statement that instructs the
+driver to change the matching document. The ``updateMany()`` method only
+updates all the documents in the collection that match the filter.
+
+To perform an update with the ``updateMany()`` method, you must pass
+a query filter and an update document. The query filter specifies the criteria
+for which document to perform the update on and the update document provides
+instructions on what changes to make to them.
+
+You can optionally pass an instance of ``UpdateOptions`` to the method in
+order to specify the behavior. For example, if you set the ``upsert`` field of
```

Comment:
```suggestion
order to adjust behavior. For example, if you set the ``upsert`` field of
```
I'm not sure if this sounds any better, but I definitely think we should at least ditch the "the" before behavior.

Reply:
Will address.